controllers/apartments_controller.rb

```diff
@@ -0,0 +1,53 @@
+require "pry"
```

Comment:
If you're `require_relative`-ing multiple files together, you actually only need to `require` Pry in one of them. If File A requires File B, and File B requires Gem C, then File A also ends up with Gem C.